Album/ViewControllers/ProfileControllers/ActivitiesViewController.m

```diff
@@ -76,6 +76,18 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
 
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
     ActivityCell *cell = [tableView dequeueReusableCellWithIdentifier:@"ActivityCell"];
+    [cell.layer setCornerRadius:20];
+    [cell setClipsToBounds:YES];
+    CAGradientLayer *grad = [CAGradientLayer layer];
```

Comment:
`gradientLayer` would be more easy to read for new developers than `grad`

Reply:
Done!